charts/cortx/templates/_config.yaml.tpl

```diff
@@ -134,36 +134,36 @@ cortx:
       services:
       - name: agent
         memory:
-          min: 128Mi
-          max: 256Mi
+          min: {{ .Values.configmap.cortxControl.agent.resources.requests.memory }}
+          max: {{ .Values.configmap.cortxControl.agent.resources.limits.memory }}
         cpu:
-          min: 250m
-          max: 500m
+          min: {{ .Values.configmap.cortxControl.agent.resources.requests.cpu }}
+          max: {{ .Values.configmap.cortxControl.agent.resources.limits.cpu }}
```

Comment:
This could benefit from a template helper.

One way could be something like (untested):

```
services:
{{- include "config.yaml.service.limits" dict ("name" "agent" "resources" .Values.configmap.cortxControl.agent.resources) | nindent 6 }}
```

Where `config.yaml.service.limits` dumps out:
```
- name: {{ .name }}
  memory:
    min: {{ .resources.requests.memory }}
    max: {{ .resources.limits.memory }}
  cpu:
    min: {{ .resources.requests.cpu }}
    max: {{ .resources.limits.cpu }}
```

Reply:
Done! Good call.